.azure-pipelines/ci.yml

```diff
@@ -339,6 +343,9 @@ steps:
 - task: PublishBuildArtifacts@1
   inputs:
     pathtoPublish: "$(Build.StagingDirectory)/envoy"
-    artifactName: ${{ parameters.ciTarget }}
+    ${{ if eq(parameters.artifactName, '') }}:
+      artifactName: ${{ parameters.ciTarget }}
+    ${{ if ne(parameters.artifactName, '') }}:
+      artifactName: ${{ parameters.artifactName }}
   timeoutInMinutes: 10
-  condition: eq(${{ parameters.publishEnvoy }}, 'true')
+  condition: and(eq(${{ parameters.publishEnvoy }}, 'true'), ne('${{ parameters.ciTarget }}', 'release.server_only'))
```

Comment:
just spotted this - which doesnt work but shouldnt be here it was a failed attempt to fix